brokers/unified/broker.go

```diff
@@ -0,0 +1,152 @@
+//
+// Copyright (c) 2018-2019 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package unified
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/eclipse/che-go-jsonrpc"
+	"github.com/eclipse/che-plugin-broker/brokers/che-plugin-broker"
+	"github.com/eclipse/che-plugin-broker/brokers/theia"
+	"github.com/eclipse/che-plugin-broker/brokers/vscode"
+	"github.com/eclipse/che-plugin-broker/common"
+	"github.com/eclipse/che-plugin-broker/model"
+	"github.com/eclipse/che-plugin-broker/storage"
+	"github.com/eclipse/che-plugin-broker/utils"
+	"net/http"
+)
+
+const ChePluginType = "Che plugin"
+const EditorPluginType = "Che Editor"
```

Comment:
`Che Editor` but `Che plugin`, is it done by purpose?

Reply:
It's a mistake. Thanks for catching